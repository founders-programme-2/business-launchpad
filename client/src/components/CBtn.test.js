import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';

const CBtn = () => {
  return <h1>Hello</h1>;
};

export default CBtn;

test('Natural', () => {
  const cbtn = renderer
    .create(
      <BrowserRouter>
        {' '}
        <CBtn />{' '}
      </BrowserRouter>
    )
    .toJSON();
  expect(cbtn).toMatchSnapshot();
});
