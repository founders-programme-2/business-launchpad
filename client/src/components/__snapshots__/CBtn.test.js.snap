// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Natural 1`] = `
Array [
  " ",
  <h1>
    Hello
  </h1>,
  " ",
]
`;
